app/moderator/moderator.js

```diff
@@ -643,13 +645,16 @@ export default class Moderator {
     }
     this.moderate(timeswitch, 'public', 'day time')
     this.narrate(`The sun rises. A new day begins for the village.`, 'public')
+
     this.resolveNightEvents();
 
     //werewolves may have won at this point
     this.checkWin();
     if (this.winner === 'werewolves'){
       let msg = `Werewolves have overrun your village and there is no hope for the innocent.`
-      this.narrate(msg, 'public', null, 'wolf win')
+      this.narrate(msg, 'public', null, 'wolf win');
+      //changes background image to show winners
+      return store.dispatch(updateWinner(this.winner));
```

Comment:
this needs to be a dispatch to firebase, using the this.moderate function and the appropriate action